source/includes/usage-examples/index-code-examples.cpp

```diff
@@ -0,0 +1,18 @@
+// start-single-field
+auto index_specification = make_document(kvp("<field name>", 1));
+std::string result = collection.create_index(std::move(index_specification));
+
+std::cout << "Index created: " << result << std::endl;
```

Comment:
```suggestion
auto result = collection.create_index(index_specification.view());
std::cout << "Index created: " << bsoncxx::to_json(result) << std::endl;
```

`create_index` returns a `bsoncxx::document::value`, not a `std::string`. Suggest using `auto` and `bsoncxx::to_json(result)` to fix compile.

Additionally, suggest using `.view()` rather than `std::move` to avoid unnecessary ownership transfer.